src/Command/Setup.php

```diff
@@ -75,7 +75,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
         return 0;
     }
 
-    private function setupDependencies(SymfonyStyle $io, MetadataBag $metadata)
+    private function setupDependencies(SymfonyStyle $io, MetadataBag $metadata) : \Generator
```

Comment:
I kept the space from what I can see in some popular repositories, but I'm looking for a standard... Do you have such?